packages/compose/src/hooks/use-focus-return/index.js

```diff
@@ -45,13 +47,15 @@ function useFocusReturn( onFocusReturn ) {
 				return;
 			}
 
-			focusedBeforeMount.current = node.ownerDocument.activeElement;
+			focusedBeforeMount.current =
+				/** @type {HTMLElement | null} */ ( node.ownerDocument
+					.activeElement ) || undefined;
 		} else if ( focusedBeforeMount.current ) {
-			const isFocused = ref.current.contains(
-				ref.current.ownerDocument.activeElement
+			const isFocused = ref.current?.contains(
+				ref.current?.ownerDocument.activeElement ?? null
```

Comment:
Can we pass `null` to `Node.contains()`? In any case this case feels weird.

Reply:
You can, it accepts `Node | null` but not `undefined`.

Reply:
👍 Gotcha. Checking if a node contains `null` sounds like an oxymoron to me, but that's beyond the scale of this PR anyway.